### PR TITLE
SITL: Add license info, remove extra include

### PR DESCRIPTION
**Pull request by rajat2004**
Just something which caught my eye, no functional changes

### CODE REVIEW

File: libraries/SITL/SIM_Scrimmage.cpp [hunk 1]
```diff
@@ -128,7 +122,7 @@ void Scrimmage::start_scrimmage(void)
         int ret = system(full_exec_str);
 
         if (ret != 0) {
-            std::cerr << "scrimmage didn't open.\n";
+            fprintf(stderr, "scrimmage didn't open.\n");
```
Comment (peterbarker) [inline libraries/SITL/SIM_Scrimmage.cpp]: Should probably be `::fprintf` here.

Comment (rajat2004) [reply libraries/SITL/SIM_Scrimmage.cpp]: Done

**Comment by rajat2004** [comment]
Had forgotten about the multiple directories thing, fixed now, thanks!

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**